torch/optim/_multi_tensor/_functional.py

```diff
@@ -299,3 +299,58 @@ def adamw(params: List[Tensor],
 
     step_size = [-1 * (lr / bc) for bc in bias_correction1]
     torch._foreach_addcdiv_(params, exp_avg, denom, step_size)
+
+
+def sgd(params: List[Tensor],
```

Comment:
Could you re-order these arguments to match the sgd() API from the non-multi-tensor one?
That should allow us to use them interchangeably which will make switching from one to the other much simpler!

Reply:
Thanks for the feedback, i did re-ordering as much as possible. But please note that, we have states and has_sparse_grad parameters here which is not existing in single tensor API.